docs/Product-Guides/Benefits-API.md

```diff
@@ -1,40 +1,253 @@
+ 5  6  7  8  9 10 11
+12 13 14 15</mark> <mark style="background-color: yellow">16</mark> <mark style="background-color: green">17</mark> 18
+19 20 21 22 23 24 25
+26 27 28 29 30
+</pre>
+
+### Couple of things to keep in mind
+
+- Often there is a company contribution and an employee contribution. Some payroll systems separate these, some add them together when calculating the deduction. When validating if the deduction is successful, make sure to understand how the provider you are working with handles contributions. Our team will add the deduction based on how the provider is set up.
+- Some payroll providers offer a dedicated payroll representative to help with making payroll changes. If a payroll rep is helping your customer’s HR admin with changes in their system, it is important that you make it explicitly clear who does what so that the payroll rep does not overwrite any changes Finch has previously made. Calling out Benefit Code types and thoughtful benefit descriptions help.
+
+## Testing benefits
+
+There are three ways we recommend testing benefits.
+
+### Option 1 - Sandbox
+
+You can [use the benefits endpoints in the Finch Sandbox](https://developer.tryfinch.com/docs/reference/ZG9jOjMxOTg1NTMw-testing#testing-benefits). This supports read operations (`GET` requests) and stubbed out write operations (`POST` requests), meaning writes will not actually change any internal state and reads will always return the same responses.
+
+### Option 2 - Automated
+
+If you need to test reading back the data you write into the system, you can use our test Gusto instance (using the company Cisticola LLC that we set up) for testing our benefits endpoints. At the time of writing (6/22/22), this Gusto instance currently has two active employees, one active contractor, and two inactive employees. The active employees can be enrolled in any of the benefits that Gusto supports. The compatibility matrices of support benefit types and benefit features can be found below - [Gusto Compatibility Matrices]().
+
+> Please keep in mind that this is a live account for a real LLC that our company runs. This means that there are tax implications to actually moving money.
+> - Please DO NOT EVER RUN PAYROLL ON THIS ACCOUNT.
+> - Please do not edit anything in the Gusto dashboard.
+> - Please do not hire or fire any employees or contractors.
+> - Please do not add any additional admins to this account.
+
+### Option 3 - Assisted
+
+Assisted Benefits via API allows you to use the Finch API to enroll employees in benefits for providers which are not yet fully automated. Because benefits work very differently across different systems, Finch offers a few test accounts to help explore assisted benefits connections. However, because assisted connections requires some manual work for our product operations team to complete the requests, we will only be able to perform a few test requests.
+
+- Zenefits
+- Paychex Flex
+- JustWorks (coming soon)
```

Comment:
Same here. Rather than listing specifics, I'd rather we just provide the information when we're setting them up
```suggestion
Assisted Benefits via API allows you to use the Finch API to enroll employees in benefits for providers which are not yet fully automated. Because benefits work very differently across different systems, Finch offers a several test accounts to help explore assisted benefits connections. Please reach out to a Finch rep to have these accounts set up.
```